keyserver/encoding.go

```diff
@@ -10,10 +10,13 @@ import (
 	. "github.com/coniks-sys/coniks-go/protocol"
 )
 
+// MarshalResponse returns the JSON encoding of the server's response.
 func MarshalResponse(response *Response) ([]byte, error) {
 	return json.Marshal(response)
 }
 
+// UnmarshalRequest parses the JSON-encoded request and
+// creates the corresponding request which could be handled by the server.
```

Comment:
s/request/protocol.message.Request (I'm not sure if this is the right convention for citing interfaces in other packages). Also, s/could/will